app/scripts/containers/config.js

```diff
@@ -0,0 +1,102 @@
+import React, {Component} from 'react'
+import {connect} from 'react-redux'
+import i18n from '../utils/i18n.js'
+
+import ConfigText from '../components/config-text'
+import ConfigButtons from '../components/config-buttons'
+import {
+  Row, Col
+}
+from 'react-bootstrap'
+
+import {
+  config as configActions,
+  pages,
+  errors
+} from '../actions'
+
+const {config} = configActions
+
+export
+default class ConfigContainer extends Component {
+  static displayName = 'Config';
+  static propTypes = {
+    // Dispatch
+    loadConfigPage: React.PropTypes.func.isRequired,
+    resetErrorMessage: React.PropTypes.func.isRequired,
+    saveConfigClick: React.PropTypes.func.isRequired,
+    saveConfigDraft: React.PropTypes.func.isRequired,
+    savedConfig: React.PropTypes.func.isRequired,
+    resetConfigDraft: React.PropTypes.func.isRequired,
+    setHeight: React.PropTypes.func.isRequired,
+    // State
+    config: React.PropTypes.string.isRequired,
+    draft: React.PropTypes.string.isRequired,
+    savingConfig: React.PropTypes.bool.isRequired,
+    saved: React.PropTypes.bool.isRequired,
+    height: React.PropTypes.bool.isRequired
+  };
+
+  componentWillMount () {
+    this.props.loadConfigPage()
+  }
+
+  componentWillUnmount () {
+    this.props.resetErrorMessage()
+    this.props.savedConfig(false)
+  }
+
+  render () {
+    const buttons = (
+      <ConfigButtons
+        resetConfigDraft={this.props.resetConfigDraft}
+        saveConfigClick={this.props.saveConfigClick}
+        savingConfig={this.props.savingConfig}
+        saved={this.props.saved}
+      />
+    )
+
+    return (
+      <Row>
+        <Col sm={10} smOffset={1}>
+          <div className='webui-config'>
+            <h3>{i18n.t('Config')}</h3>
+            <br/>
+            {buttons}
+            <div style={{height: '50px'}} />
+            <ConfigText
+              saveConfigDraft={this.props.saveConfigDraft}
+              savedConfig={this.props.savedConfig}
+              setHeight={this.props.setHeight}
+              config={this.props.config}
+              draft={this.props.draft}
+              height={this.props.height}
+            />
+            {buttons}
+            <div style={{height: '50px'}} />
+            <br/>
+          </div>
+        </Col>
+      </Row>
+    )
+  }
+}
+
+function mapStateToProps (state) {
+  return {
+    ...state.config,
+    errorMessage: state.errorMessage
+  }
+}
+
+const mapDispatchToProps = {
```

Comment:
Please put this inline in the `connect` call to be consistent with the other containers